Makefile.js

```diff
@@ -60,7 +61,7 @@ var NODE = "node ", // intentional extra space
     MAKEFILE = "./Makefile.js",
     PACKAGE = "./package.json",
     JS_FILES = find("lib/").filter(fileType("js")).join(" "),
-    JSON_FILES = find("conf/").filter(fileType("json")).join(" ") + " .eslintrc",
+    JSON_FILES = find("conf/").filter(fileType("json")),
```

Comment:
Looks like this isn't going to validate `.eslintrc` file anymore, which is also json.